### DISCUSSION TITLE
Question about Platformio.ini setup

### DISCUSSION THREAD

**Discussion by Stolowma** [luc-github/ESP3D#848] [Questions]
I'm fairly new to this so be gentle. I am about to flash a ESP-01S 4M but only listings are esp01s_160mhz-2.7.4 and esp01s_160mhz-2.5.2 and in both the board is shown as 1M. Which is the appropriate one to use to use?

**Comment by luc-github** [comment]
if your esp01 has 4MB you must use env:esp8266dev

if you are not confident you can try yo generate files with the configurator as it seems you try v3.0 https://luc-github.github.io

**Reply by Stolowma** [reply]
Ahh, I wish I had known about the configurator. I tried the 2.5.2 and changed the config to 4m2m and it seems to have worked. I got it all set up as a client but I couldn't figure out why it wouldn't communicate with the with the printer until I discovered the enable WIFI setting in the display (Duh). I'll have to test it out today.

Thanks for the advice and all your work on this.
